src/offline/cable_input.F90

```diff
@@ -483,6 +483,10 @@ SUBROUTINE open_met_file(dels,koffset,kend,spinup, TFRZ)
        IF (ok /= NF90_NOERR) CALL nc_abort &
             (ok,'Error opening netcdf met forcing file '//TRIM(filename%met)// &
             ' (SUBROUTINE open_met_file)')
+    ! R. Law (rml599gh) 30/05/24 ncid_mask needs to be set to ncid_met 
+    ! if reading a met data file otherwise code crashes if trying to do 
+    ! multiple sites with a met data file
+       ncid_mask = ncid_met
```

Comment:
Try to align the comments and the code please. 
You can directly apply this suggestion in GitHub, just click the "Commit suggestion" button.

```suggestion
       ! R. Law (rml599gh) 30/05/24 ncid_mask needs to be set to ncid_met 
       ! if reading a met data file otherwise code crashes if trying to do 
       ! multiple sites with a met data file
       ncid_mask = ncid_met
```